src/utils/coordsToScore.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * @author PhilTBatt
+ */
+export default function coordsToScore(coordinates: [number, number][]): number {
+	const earthRadiusPerDegree = 111000;
```

Comment:
OK, I'm lost already... 🤣

Reply:
I'm not sure on this function either - Phil wrote this :)

Reply:
Let's just mark as resolved and git blame. :)